src/server/pfs/server/val_server.go

```diff
@@ -118,6 +118,34 @@ func (a *validatedAPIServer) ClearCommit(ctx context.Context, req *pfs.ClearComm
 	return a.apiServer.ClearCommit(ctx, req)
 }
 
+func (a *validatedAPIServer) ForgetCommit(ctx context.Context, req *pfs.ForgetCommitRequest) (*pfs.ForgetCommitResponse, error) {
+	var resp *pfs.ForgetCommitResponse
+	return resp, errors.Wrap(a.driver.txnEnv.WithWriteContext(ctx, func(ctx context.Context, txnCtx *txncontext.TransactionContext) error {
+		c, err := pfsdb.PickCommit(ctx, req.Commit, txnCtx.SqlTx)
+		if err != nil {
+			return errors.Wrap(err, "pick commits")
```

Comment:
[nit] You should probably wrap these messages with a prefix like `forget commit:` So this would be `forget commit: pick commits`. The reasoning is that if you find the error message in a log file or output and you don't have line number and file information associated with it, you can 'resolve' where the error came from in code with that wrapped message.

Reply:
It's wrapped with a prefix "Forget Commit" on line 146